pkgs/top-level/python-packages.nix

```diff
@@ -3758,7 +3758,7 @@ in {
 
   monotonic = callPackage ../development/python-modules/monotonic { };
 
-  MySQL_python = callPackage ../development/python-modules/mysql_python { };
+  MySQL_python = callPackage ../development/python-modules/mysql_python { libmysql = pkgs.mariadb-connector-c_2_3; };
```

Comment:
Why are you inconsistently referring to the package? Shouldn't it be `libmysql` everywhere?

Reply:
With mariadb connector-c v3.1 error build
```
builder for '/nix/store/8sfw1ql6ia1wznyshylk6j69d2wyq2h7-python2.7-MySQL-python-1.2.5.drv' failed with exit code 1; last 10 log lines:
  copying MySQLdb/constants/CLIENT.py -> build/lib.linux-x86_64-2.7/MySQLdb/constants
  running build_ext
  building '_mysql' extension
  creating build/temp.linux-x86_64-2.7
  gcc -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -Dversion_info=(1,2,5,'final',1) -D__version__=1.2.5 -I/nix/store/h258lj4fc791l0lzdz04bf9za715fvxz-mariadb-connector-c-3.1.2/include/mariadb -I/nix/store/h258lj4fc791l0lzdz04bf9za715fvxz-mariadb-connector-c-3.1.2/include/mariadb/mysql -I/nix/store/whb8vd9pmssdmzhqffzkc3vz6ih3q1yk-python-2.7.16/include/python2.7 -c _mysql.c -o build/temp.linux-x86_64-2.7/_mysql.o
  _mysql.c:44:10: fatal error: my_config.h: No such file or directory
   #include "my_config.h"
            ^~~~~~~~~~~~~
  compilation terminated.
  error: command 'gcc' failed with exit status 1
[0 built (1 failed), 0.0 MiB DL]
error: build of '/nix/store/8sfw1ql6ia1wznyshylk6j69d2wyq2h7-python2.7-MySQL-python-1.2.5.drv' failed
```

Reply:
I meant why aren't we calling it `libmysql_2_3` (or more accurately `libmysqlclient_2_3`)?